spring-thymeleaf-2/src/test/java/com/baeldung/thymeleaf/mvcdata/EmailControllerUnitTest.java

```diff
@@ -0,0 +1,71 @@
+package com.baeldung.thymeleaf.mvcdata;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import com.baeldung.thymeleaf.mvcdata.repository.EmailData;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc(printOnlyOnFailure = false)
+public class EmailControllerUnitTest {
+
+    EmailData emailData = new EmailData();
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void whenCallEmailSubject_ThenReturnModelAttribute() throws Exception {
```

Comment:
For consistency with the Baeldung style guide, please call these methods "when..._then..." with a lowercase "t" instead of an uppercase "T".